src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/OpenAppManifestValueProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
+{
+    [ExportInterceptingPropertyValueProvider("CreateOrOpenAppManifest", ExportInterceptingPropertyValueProviderFile.ProjectFile)]
```

Comment:
Do we have a constant to replace the literal `"CreateOrOpenAppManifest"` string?